### ISSUE TITLE
language tabs not active when double clicking

### ISSUE THREAD

**Issue by jmartsch** [processwire/processwire-issues#1414]
Iif you double-click on a language tab, then the content of all Inputfields is changed to the selected language , but only the clicked language tab gets an active status class (uk-active).

I am using PW 3.0.179

Default language selected

![msedge_2021-07-19_14-07-41](https://user-images.githubusercontent.com/551551/126179776-b7aaabdf-5127-4265-ba9c-87fc2342973d.png)

After Double-click on english language tab
![msedge_2021-07-19_14-08-12](https://user-images.githubusercontent.com/551551/126180687-da571e91-38d1-42e0-9c2f-881f2da373ec.png)

I fixed this with mofifying the dblclickLanguageTab method in LanguageTabs.js
```
function dblclickLanguageTab(e) {
	if(clickLanguageTabActive) return;
	clickLanguageTabActive = true;
	var $tab = $(this);
	var langID = $tab.attr('data-lang');
	var $tabs = $tab.closest('form').find('a.langTab' + langID).not($tab);
	// $tabs.effect('highlight', 250);
	$links = $(document).find('.langTabLink');
	$links.each(function(){
	  $(this).parent().removeClass('uk-active');
	});

	$tab.click();
  	$tabs.click();
	$tabs.each(function(){
	  $(this).parent().addClass('uk-active');
	  $(this).parent().click();
	})
	setTimeout(function() {
		clickLanguageTabActive = false;
	}, 250);
}
```

**Comment by daun** [comment]
Can confirm on 3.0.180.

**Comment by daun** [comment]
~~Doing a hard refresh in the browser fixes the problem, so looks like it was solved in the meantime.~~

I reloaded **after** applying the above fix, which of course fixed it. Duh.

**Comment by jmartsch** [comment]
This issue is fixed in latest PW 3.0.181
